cpg_workflows/stages/joint_genotyping_qc.py

```diff
@@ -151,19 +151,19 @@ def expected_outputs(self, cohort: Cohort) -> dict[str, Path]:
         """
         if get_config()['workflow'].get('skip_qc', False) is True:
             return {}
-
+        h = get_workflow().output_version
```

Comment:
Is this the best var name?

Reply:
Nah, I really hate this name. Weirdly this was already present as `h`, and was used in the paths of all non-HTML output files. I'll improve it